internal/apis/config/webhook/types.go

```diff
@@ -27,11 +27,11 @@ type WebhookConfiguration struct {
 
 	// securePort is the port number to listen on for secure TLS connections from the kube-apiserver.
 	// Defaults to 6443.
-	SecurePort *int
+	SecurePort int32
 
 	// healthzPort is the port number to listen on (using plaintext HTTP) for healthz connections.
 	// Defaults to 6080.
-	HealthzPort *int
+	HealthzPort int32
```

Comment:
comment: I guess we can't change to `uint16` because that would remove a lot of potential invalid values from these... I'm not confident in asserting how changing from signed to unsigned would affect things though.

Reply:
I guess if it is not possible to convert the value to/ from a uint16, the conversion function could throw an error ??
But like you said, we should probably validate using our validation logic.